### PR TITLE
Feature/bib 390 return flow

### PR DESCRIPTION
**Pull request by tuj**
* Added returnMaterial flow.
* Cleaned up issues with docker setup.

### CODE REVIEW

File: assets/js/Steps/Initial.js [hunk 1]
```diff
@@ -17,20 +17,22 @@ function Initial(props) {
...
-
         const barcodeScanner = new BarcodeScanner(400);
 
         const barcodeCallback = code => {
-            console.log('barcodeCallback');
-
             // Commands are 5 characters long.
-            if (code.length <= 5) {
+            if (code.length === 5) {
                 if (code === '03009') {
```
[the remark this answers is not in the document]
Comment (sinejespersen) [reply assets/js/Steps/Initial.js:25]: perhaps also `'5'` as commandlength?

File: assets/js/Steps/ReturnMaterials.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            </Row>
+            <Row>
+                <Col>
+                    <h2>Materials</h2>
+
+                    <Table striped={true} bordered={true}>
+                        <thead>
+                            <tr>
+                                <th>ID</th>
+                                <th>title</th>
```
[the remark this answers is not in the document]
Comment (tuj) [reply assets/js/Steps/ReturnMaterials.js:63]: The string translation will be handled in a later PR.

File: assets/js/Steps/ReturnMaterials.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        return;
+    }
+
+    return (
+        <Container>
+            <h1>Return</h1>
+
+            <Row>
+                <Col>
+                    <Alert variant={'info'}>Skan materialer</Alert>
```
[the remark this answers is not in the document]
Comment (tuj) [reply assets/js/Steps/ReturnMaterials.js:52]: The string translation will be handled in a later PR.

**Comment by tuj** [comment]
Frontend tests is described here: https://github.com/bibboxen/bibbox-website#frontend-1
Translation of the frontend will be handled in a later PR since this should be configurable in the backend.

**Pull request merged**